src/createJobs.js

```diff
@@ -27,49 +23,50 @@ const fileTest = function (filename) {
  * @return {Array}
  */
 const createQaJobs = function (command, config) {
-    return [new Job(
-        command + ' on PHP ' + config.minimum_version,
+    return [ new Job(
+        `${command  } on PHP ${  config.minimumVersion}`,
         JSON.stringify(new Command(
```

Comment:
This `JSON.stringify()` will produce a data structure that is potentially quite different from what we had before :thinking:

Reply:
why do you think so? cant really follow.
JSON.stringify is already part of the latest version?

Reply:
Ah I see what you mean and yes indeed, it changes a little bit.
Good catch. We are missing to call `toJSON` here. 

There is some "normalization" stuff going on in another function which converts the string back to an object and verifies properties.
There it will not properly match anymore. 👍🏼